### PR TITLE
Preparing UG document content for Button control

### PR DESCRIPTION
**Pull request by AarthiArjunan**
In this PR I have added UG content for SfButton control.

### CODE REVIEW

**Review by Choza-rajan — APPROVED** [review]

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    HorizontalOptions = LayoutOptions.Start,
+    VerticalOptions = LayoutOptions.Center,
+    TextColor = Colors.White
+};
+
+// Add the busyIndicator and label to the stackLayout
+stackLayout.Children.Add(busyIndicator);
+stackLayout.Children.Add(label);
+
+// Return the constructed stackLayout as the root of the DataTemplate
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:79]: // Return the constructed stackLayout as the root of the DataTemplate.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+var label = new Label
+{
+    Text = "Loading...",
+    FontSize = 20,
+    HorizontalOptions = LayoutOptions.Start,
+    VerticalOptions = LayoutOptions.Center,
+    TextColor = Colors.White
+};
+
+// Add the busyIndicator and label to the stackLayout
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:75]: // Add the busyIndicator and label to the stackLayout.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    IsRunning = true,
+    TextColor = Colors.White,
+    WidthRequest = 80,
+    HeightRequest = 60,
+    IndicatorColor = Colors.Yellow,
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.End
+};
+
+// Create the Label
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:65]: // Create the Label.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+var happyTemplate = new DataTemplate(() =>
+{
+// Create the HorizontalStackLayout
+var stackLayout = new Microsoft.Maui.Controls.StackLayout()
+{
+    Orientation = StackOrientation.Horizontal,
+    VerticalOptions = LayoutOptions.Center
+};
+
+// Create the SfBusyIndicator
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:52]: // Create the SfBusyIndicator.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+{% highlight c# %}
+
+using Syncfusion.Maui.Core;
+
+. . . 
+
+SfButton button = new SfButton();
+var happyTemplate = new DataTemplate(() =>
+{
+// Create the HorizontalStackLayout
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:45]: // Create the HorizontalStackLayout.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+title: How to add the custom view for Syncfusion SfButton
+description: How to add the custom view for SfButton.
+platform: maui
+control: Sfbutton
+documentation: ug
+---
+
+# Add the custom view for button
+
+You can customize the appearance of the button by adding your custom view in the `Content` property. The following code sample demonstrates how to apply the busy indicator control as custom view for a button.
```
Comment (M-A32) [inline MAUI/Button/how-to/Add-the-custom-view-for-button.md:12]: You can customize the appearance of the button by adding your custom view in the `Content` property. The following code sample demonstrates how to apply the busy indicator control as a custom view for a button.

File: MAUI/Button/Overview.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+title: About .NET MAUI control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
+platform: maui
+control: SfButton
+documentation: ug
+---
+
+# .NET MAUI Button (SfButton) Overview
+
+The .NET MAUI Button is a custom button control with UI customization. You can set icons, background images, and corner edge radii and customize the appearance for different visual states using the visual state manager.
```
Comment (M-A32) [inline MAUI/Button/Overview.md:12]: The .NET MAUI Button is a custom button control with UI customization. Set icons, background images, and corner edge radii and customize the appearance for different visual states using the visual state manager.

File: MAUI/Button/Event.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Event in .NET MAUI Button control | Syncfusion
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Event.md:3]: Changes have been addressed.

File: MAUI/Button/Event.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+description: Learn here all about Event support in Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Event.md:4]: Changes have been addressed.

File: MAUI/Button/Event.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+            SfButton button = new SfButton();
+            button.Text = "Button";
+            button.Clicked += Button_Clicked;
+            ....
+        }
+
+        ...
+        private void Button_Clicked(object sender, EventArgs e)
+        {
+           // Invoke your desired action here
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Event.md:36]: Changes have been addressed.

File: MAUI/Button/GettingStarted.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Button | Syncfusion
+description:Learn here about getting started with Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/GettingStarted.md:4]: Changes have been addressed.

File: MAUI/Button/GettingStarted.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+
+This section explains the steps required to work with the `.NET MAUI Button` control.
+
+## Creating an application with .NET MAUI Button
+
+This section explains the steps required to work with the `SfButton` control for .NET MAUI.
+
+## Adding a .NET MAUI Button reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI Button` to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Buttons`, and install it.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/GettingStarted.md:20]: Changes have been addressed.

File: MAUI/Button/Migration.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI Button | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin Button to Syncfusion .NET MAUI Button control and more here.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Migration.md:4]: Changes have been addressed.

File: MAUI/Button/Migration.md [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+platform: maui
+control: SfButton
+documentation: ug
+---  
+
+# Migrating from Xamarin SfButton to .NET MAUI SfButton 
+
+To migrate easier from Xamarin SfButton to `.NET MAUI SfButton`, we kept most of the APIs from Xamarin SfButton in MAUI SfButton. However, to maintain the consistency of API naming in MAUI SfButton, we renamed some of the APIs. The APIs changed in MAUI SfButton from Xamarin SfButton are detailed as follows:
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Migration.md:12]: Changes have been addressed.

File: MAUI/Button/Overview.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: About .NET MAUI control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Overview.md:4]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: How to add the custom view for Syncfusion SfButton
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:3]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 2 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:75]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 3 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:65]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 4 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:52]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 5 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:45]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 6 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:12]: Changes have been addressed.

File: MAUI/Button/Overview.md (hunk 7 above)
Comment (AarthiArjunan) [reply MAUI/Button/Overview.md:12]: Changes have been addressed.

File: MAUI/Button/how-to/Add-the-custom-view-for-button.md (hunk 1 above)
Comment (AarthiArjunan) [reply MAUI/Button/how-to/Add-the-custom-view-for-button.md:79]: Changes have been addressed.

File: MAUI/Button/Overview.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: About .NET MAUI control | Syncfusion
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Button/Overview.md:3]: Changes have been addressed.

File: MAUI/Button/Customization.md [hunk 18]
```diff
@@ -0,0 +1,671 @@
...
+documentation: ug
+---
+
+# Customization in .NET MAUI Button (SfButton)
+
+The `.NET MAUI Button` control supports to customize the border color, image width, corner radius, background color, and more. The button control can be customized using the following properties:
+
+## Text Customization
+
+The text inside the button can be customized by its text color, font size, font attributes, font family and text alignment.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:16]: The text inside the button can be customized by its text color, font size, font attributes, font family, and text alignment.

File: MAUI/Button/Customization.md [hunk 19]
```diff
@@ -0,0 +1,671 @@
...
+
+### TextColor
+
+The `TextColor` property is used to customize the color of text in `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:20]: The `TextColor` property is used to customize the color of the text in `SfButton`.

File: MAUI/Button/Customization.md [hunk 20]
```diff
@@ -0,0 +1,671 @@
...
+button.TextColor = Color.White;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with text color](images/Button_textcolor.png)
+
+### FontSize
+
+The `FontSize` property is used to customize the size of text in `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:42]: The `FontSize` property is used to customize the size of the text in `SfButton`.

File: MAUI/Button/Customization.md [hunk 21]
```diff
@@ -0,0 +1,671 @@
...
+button.Text = "Button";
+button.HorizontalTextAlignment = TextAlignment.Center;
+button.VerticalTextAlignment = TextAlignment.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Background Customization
+
+The background of the button can be customized by its background color, border color, border width and corner radius.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:129]: The background of the button can be customized by its background color, border color, border width, and corner radius.

File: MAUI/Button/Customization.md [hunk 22]
```diff
@@ -0,0 +1,671 @@
...
+button.Background = Colors.DeepSkyBlue;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with background color](images/Button_backgroundcolor.png)
+
+### Stroke
+
+The `BorderColor` property is used to customize the color of border in `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:155]: The `BorderColor` property is used to customize the color of the border in `SfButton`.

File: MAUI/Button/Customization.md [hunk 23]
```diff
@@ -0,0 +1,671 @@
...
+button.Stroke = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with stroke](images/Button_border.png)
+
+### StrokeThickness
+
+The `BorderWidth` property is used to customize the thickness of border in `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:177]: The `BorderWidth` property is used to customize the thickness of the border in `SfButton`.

File: MAUI/Button/Customization.md [hunk 24]
```diff
@@ -0,0 +1,671 @@
...
+
+![SfButton with cornerradius](images/Button_cornerradius.png)
+
+## Image Customization
+
+The Image can be customized by its ShowIcon, ImageSource, ImageSize and ImageAlignment.
+
+### ShowIcon
+
+You can enable the Icon image using the `ShowIcon` property to know whether any image appears to the `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:226]: You can enable the Icon image using the `ShowIcon` property to know whether any image appears in the `SfButton`.

File: MAUI/Button/Customization.md [hunk 25]
```diff
@@ -0,0 +1,671 @@
...
+button.ShowIcon = True;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with content](images/Button_icon.png)
+
+### ImageSize
+
+The `ImageSize` property is used to customize the width of icon image in `SfButton`.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:272]: The `ImageSize` property is used to customize the width of the icon image in `SfButton`.

File: MAUI/Button/Customization.md [hunk 26]
```diff
@@ -0,0 +1,671 @@
...
+button.ImageSource = "Heart.png";
+button.ShowIcon = true;
+button.ImageSize= 50;
+
+{% endhighlight %}
+{% endtabs %}
+
+### ImageAlignment 
+
+The `ImageAlignment` property is used to customize the alignment of icon image in `SfButton`. The following options are available in `ImageAlignment`:
```
Comment (M-A32) [inline MAUI/Button/Customization.md:296]: The `ImageAlignment` property is used to customize the alignment of the icon image in `SfButton`. The following options are available in `ImageAlignment`:

File: MAUI/Button/Customization.md [hunk 27]
```diff
@@ -0,0 +1,671 @@
...
+
+* `Start` - Places the image at the left most of `SfButton`.
+* `End` - Places the image at the right most of `SfButton`.
+* `Top` - Places the image at the top of the text.
+* `Bottom` - Places the image at the bottom of the text.
+* `Left` - Although the flow direction has been applied, it always places the image in the left part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the right. Use `Left` alignment to show this in the same left position.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:302]: * `Left` - Although the flow direction has been applied, it always places the image in the left part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the right. Use the `Left` alignment to show this in the same left position.

File: MAUI/Button/Customization.md [hunk 28]
```diff
@@ -0,0 +1,671 @@
...
+* `Right` - Although flow direction has been applied, the image is always located in the right part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the left. But use `Right` alignment to show this in the same right position.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:303]: * `Right` - Although flow direction has been applied, the image is always located in the right part of `SfButton`. For example, in the direction of the RTL flow, the image setting will move to the left. But use the `Right` alignment to show this in the same right position.

File: MAUI/Button/Customization.md [hunk 29]
```diff
@@ -0,0 +1,671 @@
...
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with top alignment](images/Button_imagealignment_bottom.png)
+
+**Left image alignment in `SfButton`**
+
+In RTL flow direction, image alignment with `Start` will change its direction of placing image to the right. To keep that in same left position, set `Left` alignment as shown in the following code sample.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:460]: In the RTL flow direction, image alignment with `Start` will change its direction of placing the image to the right. To keep that in the same left position, set the `Left` alignment as shown in the following code sample.

File: MAUI/Button/Customization.md [hunk 30]
```diff
@@ -0,0 +1,671 @@
...
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with left alignment](images/Button_imagealignment_left.png)
+
+**Right image alignment in `SfButton`**
+
+In RTL flow direction, image alignment with `End` will change its direction of placing image to the left. To keep that in same right position, set `Right` alignment as shown in the following code sample.
```
Comment (M-A32) [inline MAUI/Button/Customization.md:500]: In the RTL flow direction, image alignment with `End` will change its direction of placing the image to the left. To keep that in the same right position, set the `Right` alignment as shown in the following code sample.

File: MAUI/Button/Customization.md [hunk 31]
```diff
@@ -0,0 +1,671 @@
...
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with right alignment](images/Button_imagealignment_right.png)
+
+## Gradient background
+
+You can set the gradient as background of SfButton using the `Background` property. It supports the following types of gradients:
```
Comment (M-A32) [inline MAUI/Button/Customization.md:540]: You can set the gradient as the background of SfButton using the `Background` property. It supports the following types of gradients:

File: MAUI/Button/Event.md (hunk 8 above)
Comment (M-A32) [inline MAUI/Button/Event.md:3]: title: Event in .NET MAUI Button Control | Syncfusion

**Review by M-A32 — COMMENTED** [review]
I have completed the review.

**Pull request merged**